addon/mixins/axis-titles.js

```diff
@@ -11,10 +11,31 @@ import Ember from 'ember';
  */
 const AxisTitlesMixin = Ember.Mixin.create({
   /**
-   * Toggle axis titles on/off
+   * Toggle axis X title on/off
    * @type {Boolean}
    */
-  hasAxisTitles: false,
+  hasXAxisTitle: false,
+
+  /**
+   * Toggle axis Y title on/off
+   * @type {Boolean}
+   */
+  hasYAxisTitle: false,
+
+  /**
+   * A deprecated property to support backward compatability.
+   * @deprecated
+   */
+  hasAxisTitles: Ember.computed('hasXAxisTitle', 'hasYAxisTitle', function() {
+    if (arguments.length > 1) {
+      // Setter case.
+      this.set('hasXAxisTitle', true);
+      this.set('hasYAxisTitle', true);
+      return true;
+    }
+
+    return this.get('hasXAxisTitle') || this.get('hasYAxisTitle');
```

Comment:
This should probably be `&&` as you may want `hasXAxisTitles` but not `hasYAxisTitles` and if you have one but not the other you probably shouldn't set `hasAxisTitles` to true.